### PR TITLE
bionic change

### PR DESCRIPTION
**Pull request by gauthamm**
Check bionic change

### CODE REVIEW

**Comment by sumandeepb** [comment]
@gauthamm the rr-devel branch is updated to v2.2.9. Now the CI config script work can be taken up

File: .travis.yml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-  # - sudo apt-key adv --keyserver keys.gnupg.net --recv-key C8B3A55A6F3EFCDE || sudo apt-key adv --keyserver hkp://keys.gnupg.net:80 --recv-key C8B3A55A6F3EFCDE
-  - sudo add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo xenial main"
+  - sudo apt-key adv --keyserver keys.gnupg.net --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-key C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver keys.gnupg.net --recv-key C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C8B3A55A6F3EFCDE
+  - sudo add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo bionic main" -u
   - sudo apt-get update -qq
   - sudo apt-get install librealsense2-dkms --allow-unauthenticated -y
```
Comment (sumandeepb) [inline .travis.yml]: should we fix the librealsense2 version to 2.29.0 ?

File: .travis.yml [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
   - sudo apt-get install librealsense2-dev --allow-unauthenticated -y
```
Comment (sumandeepb) [inline .travis.yml]: Also do we need to add librealsense2 and also perhaps librealsense2-gl ?

Comment (sumandeepb) [reply .travis.yml]: Basically something similar to https://bitbucket.org/rapyutians/sootballs/src/devel/sensors/app/scripts/install.sh

Comment (gauthamm) [reply .travis.yml]: Seems like realsense2_camera does not need it. Maybe they are needed by the utils for actual realsense hardware for display purposes.

File: .travis.yml (hunk 1 above)
Comment (gauthamm) [reply .travis.yml]: Yeah I will add it here.

File: .travis.yml (hunk 2 above)
Comment (sumandeepb) [reply .travis.yml]: OK. I see.

**Comment by gauthamm** [comment]
As discussed will merge this configuration changes in, the failing CI tests will be done in another PR.

**Pull request merged**